simpleami.c

```diff
@@ -36,6 +36,8 @@
 static void simple_callback(struct ami_session *ami, struct ami_event *event)
 {
 	const char *eventname = ami_keyvalue(event, "Event");
+	(void) ami;
```

Comment:
As part of this change, could you also modify lines 62 and 63 of simpleami.c? The 1st argument should be NULL, forgot to update those as originally cami didn't support multiple sessions.

Reply:
Done